client/signup/steps/plans-theme-preselected/helper/index.ts

```diff
@@ -0,0 +1,38 @@
+import {
+	MARKETPLACE_THEME,
+	DOT_ORG_THEME,
```

Comment:
A community theme and a DOT_ORG_THEME should mean the same thing - a theme that's pulled from WordPress.org.

Reply:
Are you suggesting removing the DOT_ORG_THEME checking?

I saw we don't have a reference to `DOT_ORG` on the endpoint we are using to pull the theme list, but I am not sure if there is any other entry point the user can use to access the `copy-site` flow that is using "dot-org" instead of "community."

Since it is a blocker issue, I think we can investigate it better in another task.